README.md

```diff
@@ -1,73 +1,35 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
+# Portfolio: setup and mobile version skeleton
 
+   > A program for setting up a skeleton  for website for a mobile version
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+**This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing**
 
```

Comment:
- [ ] In this section, please give a directive on how someone can get the project onto their computer working. You can give instructions on how to clone the project, how to run it and all other steps one can go through to make use of the repo. 👍